internal/pkg/manifest/validate.go

```diff
@@ -44,7 +44,26 @@ var (
 )
 
 // Validate returns nil if LoadBalancedWebService is configured correctly.
-func (l *LoadBalancedWebService) Validate() error {
+func (l LoadBalancedWebService) Validate() error {
```

Comment:
Because it won't work otherwise since `Validate()` would be the method of `* LoadBalancedWebService`. Also I feel it doesn't make the code much cleaner with pointer receiver.